api/v1alpha3/machineset_webhook.go

```diff
@@ -78,3 +86,29 @@ func (m *MachineSet) validate() error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineSet").GroupKind(), m.Name, allErrs)
 }
+
+// PopulateDefaultsMachineSet fills in default field values.
+// This is also called during MachineSet sync.
+func PopulateDefaultsMachineSet(ms *MachineSet) {
```

Comment:
Can you add a test for this, similar to the one we have for the MachineDeployment defaulting webhook? https://github.com/kubernetes-sigs/cluster-api/blob/master/api/v1alpha3/machinedeployment_webhook_test.go#L27-L48

Reply:
yup! got it.